news/news_win_fix_blocked_downloads.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* <news item>
+
+**Deprecated:**
+
+* <news item>
+
+**Removed:**
+
+* <news item>
+
+**Fixed:**
+
+* Avoid virus scanners blocking downloads (#2299)
```

Comment:
```suggestion
* Use ``powershell`` instead of ``certutil``to avoid virus scanners blocking downloads (#2299)
```

Reply:
Thanks @xhochy for the feedback. I committed your suggestion.